src/os/filestore/FileJournal.cc

```diff
@@ -1609,8 +1609,6 @@ void FileJournal::submit_entry(uint64_t seq, bufferlist& e, uint32_t orig_len,
   assert(e.length() > 0);
   assert(e.length() < header.max_size);
 
-  if (osd_op)
-    osd_op->mark_event("commit_queued_for_journal_write");
```

Comment:
hi @tchaikov would you mind taking a look? should we delete this event directly or use other event name?  thank you

Reply:
i think we'd better remove this one. as their timestamp should always be identical.